packages/sdk/connect/readme.md

```diff
@@ -0,0 +1,13 @@
+# @celo/connect
+
+
+## Example
+
+```javascript
+const web3 = new Web3("YOUR_RPC_URL")
+const connection = new Connection(web3)
+
+const connectedChainID =  await connection.chainId()
```

Comment:
Start of Readme for Connect package.

Reply:
PS Ive moved READme updates to a separate Branch as this one is A bit large already